lib/samples/apply_simple_renderer_to_feature_layer/README.metadata.json

```diff
@@ -2,10 +2,22 @@
     "category": "Visualization",
     "description": "Change the appearance of a feature layer with a renderer.",
     "ignore": false,
-    "images": [],
-    "keywords": [],
-    "redirect_from": [],
-    "relevant_apis": [],
-    "snippets": [],
+    "images": [
+        "apply_simple_renderer_to_feature_layer.png"
+    ],
+    "keywords": [
+        "feature layer",
+        "renderer",
+        "visualization"
```

Comment:
Keywords should contain both the relevant apis as well as the tags. There's a script you can run to generate this - I'll add the details in the issue.